src/Concordium/Client/Runner.hs

```diff
@@ -2806,6 +3318,9 @@ processLegacyCmd action backend =
     GetIdentityProviders block -> withClient backend $ withBestBlockHash block getIdentityProviders >>= printJSON
     GetAnonymityRevokers block -> withClient backend $ withBestBlockHash block getAnonymityRevokers >>= printJSON
     GetCryptographicParameters block -> withClient backend $ withBestBlockHash block getCryptographicParameters >>= printJSON
+    GetChainParameters -> do
+      bs <- withClient backend $ getFromJson =<< withBestBlockHash Nothing getBlockSummary
+      Queries.bsWithUpdates bs $ const $ printJSONValues . toJSON . Types._currentParameters
```

Comment:
Why was this added? It does not really fit the rest of the raw commands which just forward the response.

It would be great if there was a high-level command for this, but this one should be removed I think.